### PR TITLE
enable prod deploys (just via circle-ci for now)

### PR DESCRIPTION
**Pull request by cmharlow**
Jira: https://mozilla-hub.atlassian.net/browse/DSRE-637

What this PR does:
* adds firebase prod rally-web app json config file
* updates circleci deployment baseline to also now deploy prod => members.rally.mozilla.org
* updates circleci deployment note of possibly simpler to remember stage domain => members.rally.allizom.org

Next steps:
* still setting up the clickops stuff in prod, but ready for the team to start reviewing and poking at, perhaps try an automating deploy to

prod: members.rally.mozilla.org
stage (feel free to continue to use other domain too): members.rally.allizom.org (cert still created at time of writing this PR, fyi)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -117,7 +117,7 @@ workflows:
             branches:
               only: master
 
-      - site-deploy: # This deploys to our stage site at https://stage.rally-web.nonprod.dataops.mozgcp.net
+      - site-deploy: # This deploys to our stage site at https://stage.rally-web.nonprod.dataops.mozgcp.net / https://members.rally.allizom.org
```
Comment (jepstein) [inline .circleci/config.yml:120]: I assume this intentional obfuscation. I'd not seen this particular domain before.

Comment (cmharlow) [reply .circleci/config.yml:120]: allizom.org? its used by some teams (and not others) for mozilla stage (you probably already noticed, its just mozilla spelled backwards). I find it easier to remember myself than the env.name.realm.team.mozgcp.net guy (for whatever reason).

Comment (cmharlow) [reply .circleci/config.yml:120]: (i will happily remove this though if you prefer not to have it)

Comment (jepstein) [reply .circleci/config.yml:120]: nah. keep it.

**Review by jepstein — APPROVED** [review]

**Review by rhelmer — APPROVED** [review]

**Pull request merged**